systest/device_version_maps.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+export BIGIP_11_5_4="bigip-osready-11.5.4.2.0.291"
+export BIGIP_11_6_0="bigip-osready-11.6.0.0.0.401"
+export BIGIP_11_6_1="bigip-osready-11.6.1.1.0.326"
+export BIGIP_12_1_1="bigip-osready-12.1.1.2.0.204"
```

Comment:
Looks like only the 11.6.1 image exists publicly in the lab openstack deployment.  All images must be present before merging this PR.